src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+    if (days === 7) {
+    return 230;
```

Comment:
The condition checks for exactly 7 days, which does not account for rentals longer than 7 days. The discount should apply for 7 or more days.